packages/webapp/src/containers/Map/index.js

```diff
@@ -217,7 +183,9 @@ export default function Map() {
       dispatch(setMapFilterSetting(payload));
     }
   };
-  const handleAddMenuClick = (asset) => {};
+  const handleAddMenuClick = (asset) => {
+    startDrawing(asset);
```

Comment:
Perhaps we can rename `asset` to something like `locationType`? might be more clear

Reply:
same with `handleFilterMenuClick` above